src/_includes/core-nodes/read-file-use-case.md

```diff
@@ -0,0 +1,59 @@
+## What is the Read File node in Node-RED?
+
+The Read File node in Node-RED is used to read the contents of a file from the file system. It can read both string and binary buffer data, making it versatile for various integration needs within Node-RED flows.
```

Comment:
You should explain how FlowFuse currently does this differently on devices versus in the cloud